tasks/build-fits-site.yml

```diff
@@ -0,0 +1,83 @@
+- name: Build site for FITS microservice
+  blockinfile:
+    path: "/etc/apache2/conf-available/CrayFits.conf"
+    create: yes
+    block: |
+      Alias "/crayfits" "/var/www/html/CrayFits/public"
+      <Directory "/var/www/html/CrayFits/public">
+        FallbackResource /index.php
+        Require all granted
+        DirectoryIndex index.php
+        SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
+      </Directory>
+
+- name: Enable FITS site
+  file:
+    src: "/etc/apache2/conf-available/CrayFits.conf"
+    dest: "/etc/apache2/conf-enabled/CrayFits.conf"
+    state: link
+
+- name: Change owner on html dir
+  file:
+    path: /var/www/html
+    owner: www-data
+    group: www-data
+
+- name: Download islandora FITS module
+  composer:
+    command: require
+    arguments: islandora-rdm/islandora_fits
+    working_dir: "{{ drupal_core_path }}/.."
+  become_user: "{{ ansible_user }}"
+
+- name: Install Islandora FITS module
+  command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_fits"
+
+- name: Install FITS Microservice from Github
+  git:
+    repo: https://github.com/roblib/CrayFits.git
+    dest: /var/www/html/CrayFits
+    force: yes
+
+- name: Clean Directory (Failure okay)
+  file:
+    path: '/var/www/html/CrayFits/vendor/symfony/monolog-bundle'
+  with_items:
+    - absent
+    - directory
+  ignore_errors: yes
+
+- name: Change CrayFits directory ownership, group and permissions
+  file:
+    path: /var/www/html/CrayFits
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    mode: '0755'
+    recurse: True
+
+- name: Open file for logging
+  file:
+    path: /var/www/html/CrayFits/var
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    mode: '0777'
+    recurse: True
+
+- name: Restart apache
+  service:
+    name: apache2
+    state: restarted
+  become: true
+
+- name: Check for Existing microservice
+  stat:
+    path: /var/www/html/CrayFits/vendor
+  register: crayfits
+
+- name: Run Composer On Fits Microservice
+  composer:
+    command: update
+    working_dir: /var/www/html/CrayFits
```

Comment:
Annnnnnd use yet to be created FITS install directory variable. Also as a TODO, we need to commit the lock file and then just do `composer install` to ensure everyone installs the same version of the dependencies.